features/step_definitions/basic_steps.rb

```diff
@@ -568,3 +568,11 @@
   expect(stderr).not_to include "Error"
   expect(status).to be_success
 end
+
+Then(/^I should have a page with a title: "([^"]*)"$/) do |title|
```

Comment:
I guess we could adjust this to indicate that we mean a title tag in the head of the html document, so as not to confuse it with displaying an h1 title ...